contrib/adaptive-compression/test-performance.sh

```diff
@@ -0,0 +1,59 @@
+echo "testing time -- no limits set"
```

Comment:
Most of these tests are redundant. They don't bring a lot of additional value.
As a general rule, find a way to reduce time spent in tests. The faster, the better.

Also : here would be a good place to introduce rate-limited tests